commander/efd.py

```diff
@@ -90,16 +91,84 @@ async def query_efd_timeseries(request):
         response_data = dict(zip(sources, results))
         return web.json_response(response_data)
 
+    async def query_efd_logs(request):
+        global efd_clients
+        req = await request.json()
+
+        try:
+            efd_instance = req["efd_instance"]
+            start_date = req["start_date"]
+            end_date = req["end_date"]
+            cscs = req["cscs"]
+        except Exception:
+            return web.json_response(
+                {"ack": "Some of the required parameters is not present"}, status=400
+            )
+
+        efd_client = efd_clients.get(efd_instance)
+        if efd_client is None:
+            efd_client = connect_to_efd_intance(efd_instance)
+        if efd_client is None:
+            return unavailable_efd_client()
+
+        start_date = Time(start_date, scale="utc")
+        end_date = Time(end_date, scale="utc")
+        query_tasks = []
+        sources = []
+        for csc in cscs:
+            indices = cscs[csc]
```

Comment:
Maybe keep the name of the variables in English?

Reply:
This is already in English jeje